index.html

```diff
@@ -8,15 +8,50 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
     <link rel="stylesheet" href="assets/css/style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&family=Sora:wght@200;@300&display=swap">
```

Comment:
For the google fonts to display you need to put the stylesheet about the <link rel="stylesheet" href="assets/css/style.css">.  The <link rel="stylesheet" href="assets/css/style.css"> should be the very last link added in the head.  When you use google fonts you have to add all of the links they give you, not just the last one.  So to get the fonts to display you would add this above the link you already added, `<link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>`